pandas/tests/tools/test_to_datetime.py

```diff
@@ -1098,7 +1107,7 @@ def test_datetime_invalid_scalar(self, value, format, warning):
 
     @pytest.mark.parametrize("value", ["3000/12/11 00:00:00"])
     @pytest.mark.parametrize(
-        "format,warning", [(None, UserWarning), ("H%:M%:S%", None)]
+        "format,warning", [(None, UserWarning), ("%H:%M:%S", None)]
     )
     def test_datetime_outofbounds_scalar(self, value, format, warning):
```

Comment:
the test claims to be testing out-of-bounds scalars, but in reality it was erroring before the parsing even started because `"H%:M%:S%"` is an invalid format

So, I've changed the format to reflect what it looks like this test was meant to be testing, and have made a separate test below to check that invalid formats raise